src/Api/PubnubApi/EventEngine/Subscribe/Events/SubscriptionEvents.cs

```diff
@@ -4,7 +4,6 @@ namespace PubnubApi.PubnubEventEngine.Subscribe.Events {
 	public class SubscriptionChangedEvent : Core.IEvent {
 		public IEnumerable<string> Channels;
 		public IEnumerable<string> ChannelGroups;
-		public SubscriptionCursor Cursor;
```

Comment:
Cursor is removed?

Reply:
Yes, according to the docs. At least "for now".